_posts/2017-05-11-node-tech.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title: node.tech Meeting
+slug: node-tech-night
```

Comment:
If you know if it a standing meeting and it is open for people to join, why wouldn't you have them in?